projects/igniteui-angular/src/lib/directives/dragdrop/dragdrop.directive.ts

```diff
@@ -62,60 +62,181 @@ export class IgxDropEventArgs {
 })
 export class IgxDragDirective implements OnInit, OnDestroy {
 
+    /**
+     * @hidden
+     */
     @Input('igxDrag')
     public data: any;
```

Comment:
This shouldn't be hidden. This can be used to save data inside the igxDrag directive and can be set like and initialize igxDrag at the same time:
```
<div [igxDrag]="{}"></div>
```